chainercv/datasets/voc/voc_bbox_dataset.py

```diff
@@ -9,7 +9,7 @@
 from chainercv.utils import read_image
 
 
-class VOCDetectionDataset(chainer.dataset.DatasetMixin):
+class VOCBboxDataset(chainer.dataset.DatasetMixin):
 
     """Dataset class for the detection task of PASCAL `VOC`_.
```

Comment:
`Dataset class for the detection task` is intended? This description is true because it is designed for detection. However, this is against the concept of separating dataset and tasks.

Reply:
>  this is against the concept of separating dataset and tasks.

Right. I will fix this.

Reply:
I found `VOCSemanticSegmentationDataset` also has task-based description. https://github.com/chainer/chainercv/blob/master/chainercv/datasets/voc/voc_semantic_segmentation_dataset.py#L12
 Can you fix it?

Reply:
Thanks.
I made another PR.